src/app/services/scheduler/scheduler.service.ts

```diff
@@ -87,8 +87,19 @@ export class SchedulerService {
             if (allocations && allocations.length > 0) {
               const appAllocations = [];
               allocations.forEach(alloc => {
+                if (
+                  alloc.allocationTags['kubernetes.io/meta/namespace'] &&
+                  alloc.allocationTags['kubernetes.io/meta/podName']
+                ) {
+                  alloc[
+                    'displayName'
+                  ] = `${alloc.allocationTags['kubernetes.io/meta/namespace']}/\r${alloc.allocationTags['kubernetes.io/meta/podName']}`;
+                } else {
+                  alloc['displayName'] = `some-namespace/undefined`;
```

Comment:
@kobe860219 It does not make sense to display `some-namespace/undefined` when above specific allocationTags are not available. Could you pls change this to `n/a` may be?